src/components/multiAutoCompleteInput.js

```diff
@@ -150,6 +150,9 @@
         ) {
           return parsed.map((obj) => String(obj.id));
         }
+        if (typeof parsed === 'number') {
+          return [parsed];
```

Comment:
Parsed?

Reply:
agree that this name isn't great. I've changed it to parsedValue in latest commit, so it makes a bit more sense.